resources/views/account/change-password.blade.php

```diff
@@ -11,7 +11,7 @@
 
 <div class="row">
     <div class="col-md-9">
-    {{ Form::open(['method' => 'POST', 'files' => true, 'class' => 'form-horizontal', 'autocomplete' => 'off']) }}
+    <form method="POST" action="{{ route('account.password.update') }}" accept-charset="UTF-8" class="form-horizontal" autocomplete="off" enctype="multipart/form-data">
```

Comment:
We don't need multipart here - no file upload.

Reply:
Good point. Removed.